pretix_zugferd/views.py

```diff
@@ -48,3 +50,27 @@ def get_success_url(self) -> str:
                 "event": self.request.event.slug,
             },
         )
+
+    def get_context_data(self, **kwargs):
+        return super().get_context_data(
+            **kwargs,
+            is_zugferd_renderer=isinstance(
+                self.request.event.invoice_renderer, ZugferdMixin
+            ),
+            is_xrechnung_renderer=getattr(
+                self.request.event.invoice_renderer, "profile", None
+            )
+            == "XRECHNUNG",
```

Comment:
(not important but I had to look at this three times to understand it belongs together)
```suggestion
            ) == "XRECHNUNG",
```

Reply:
thank you for nothing, black…